### DISCUSSION TITLE
Data vs [UInt8]

### DISCUSSION THREAD

**Discussion by dimitribouniol** [swift-server/webauthn-swift#38] [General]
It seems like `Data` and `[UInt8]` byte arrays are applied rather inconsistently. Would we be opposed to unifying under `[UInt8]` arrays and slices whenever possible, only wrapping to `Data` when `CryptoKit` or `JSONEncoder`/`JSONDecoder` require it? Happy to submit PRs for this.

**Comment by 0xTim** [comment]
Yeah I'm happy to migrate to `[UInt8]` until we have a central 'bag of bytes' in the ecosystem
